.circleci/config.yml

```diff
@@ -75,10 +75,11 @@ references:
           DATABASE_URL: "postgres://postgres@localhost/hmpps-book-secure-move-api"
           EXTERNAL_URL: mocked_in_tests
           ENCRYPTOR_SALT: "*\xDB\x9E\a\x1ES1\xB7\xE3\xB8{\xF9Fl\x8Fnn?\x83\xC6\xBBR\x9Eq\x0F\x01./U\x0E\xF1\xDE"
-      - image: circleci/postgres:10.4
+      - image: circleci/postgres:10-alpine-ram
```

Comment:
not familiar with with `RAM` images, how does this make the build faster?

Reply:
I believe the ram images store the database files on a ram disk (instead of an hdd / ssd), so in theory, might be a little faster.

Reply:
and the alpine image is smaller (26mb vs 80mb) so should load a little faster